masterlist.yaml

```
@@ -6430,39 +6430,41 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 103
 
-  - name: 'Enhanced Vegetation \[\d{3}%\]\.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/23783/' ]
+  - name: 'Enhanced Vegetation( \[\d{3}%\])?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/23783/'
+        name: 'Enhanced Vegetation'
+      - link: 'https://www.nexusmods.com/oblivion/mods/52253/'
+        name: 'Enhanced Vegetation repack'
     msg:
       - <<: *useOnlyOneX
-        subs: [ 'Enhanced Vegetation' ]
-        condition: 'many("Enhanced Vegetation \[\d{3}%\]\.esp")'
-  - name: 'Enhanced Vegetation [100%].esp'
+        subs: [ 'Enhanced Vegetation ESP' ]
+        condition: 'many("Enhanced Vegetation( \[\d{3}%\])?\.esp")'
     clean:
```

Instead of applying the cleaning data to every plugin & therefore incorrectly applying it to some, it can just be dropped for the old plugins & applied to the new, renamed one:
```suggestion
  - name: 'Enhanced Vegetation.esp'
    clean:
```